.github/workflows/daily-build.yml

```diff
@@ -0,0 +1,57 @@
+name: Daily build
+
+# Controls when the action will run.
+on: 
+  schedule:
+    - cron: '30 2 * * *'
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+    # This workflow contains a single job called "build"
+    build:
+        # The type of runner that the job will run on
+        runs-on: ubuntu-latest
+
+        # Steps represent a sequence of tasks that will be executed as part of the job
+        steps:
+            # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+            - uses: actions/checkout@v2
+
+            # Set up Java Environment
+            - name: Set up JDK 11
+              uses: actions/setup-java@v1
+              with:
+                java-version: 11        
+            
+            # Build the ballerina project
+            - name: Ballerina Build
+              uses: ballerina-platform/ballerina-action@nightly
+              with:
+                  args:
+                      pack
+              env:
+                JAVA_HOME: /usr/lib/jvm/default-jvm
+
+            # Test the ballerina project
+            - name: Ballerina Test
+              uses: ballerina-platform/ballerina-action@nightly
+              with:
+                  args:
+                      test
+              env:
+                JAVA_HOME: /usr/lib/jvm/default-jvm
+
+            # Send notification when build fails
+            - name: Notify failure
+              if: ${{ failure() }}
+              run: |
+                curl -X POST \
+                'https://api.github.com/repos/ballerina-platform/ballerina-release/dispatches' \
+                -H 'Accept: application/vnd.github.v3+json' \
+                -H 'Authorization: Bearer ${{ secrets.BALLERINA_BOT_TOKEN }}' \
+                --data "{
+                  \"event_type\": \"notify-build-failure\",
+                  \"client_payload\": {
+                    \"repoName\": \"module-ballerinax-snowflake.driver\"
```

Comment:
Shall we add correct repo name here?

Reply:
Resolved in [f2b88ef](https://github.com/ballerina-platform/module-ballerinax-cdata.connect.driver/pull/1/commits/f2b88ef558ec076fcda27693a6f77f0a52948412). Thank you.